### PR TITLE
#109: rating, points and feedbackRating properties in several ViewMod…

### PR DESCRIPTION
**Pull request by KhojiakbarFarkhadov**
* Worked with viewModels
* Modified in cshtml files

### CODE REVIEW

File: Expro/ViewModels/AppUserVM.cs [hunk 1]
```diff
@@ -76,12 +76,9 @@ public bool IsAdmin
 
         public bool IsOnline { get; set; }
         public string DateLastSeen { get; set; }
-
-        [Display(Name = "Балл")]
-        public int Points { get; set; }
-        [Display(Name = "Рейтинг")]
-        public int Rating { get; set; }
-
+        public List<PointsInfo> PointsInfo { get; set; }
```
Comment (napster1991) [inline Expro/ViewModels/AppUserVM.cs:79]: bu propertyni ma'nosi nima?

File: Expro/ViewModels/AppUserVM.cs [hunk 2]
```diff
@@ -76,12 +76,9 @@ public bool IsAdmin
...
+        public List<RatingsInfo> RatingsInfo { get; set; }
```
Comment (napster1991) [inline Expro/ViewModels/AppUserVM.cs:80]: bu propertyni ma'nosi nima?

File: Expro/ViewModels/AppUserVM.cs [hunk 3]
```diff
@@ -217,8 +208,9 @@ public AppUserVM(ApplicationUser user)
                 user.DateLastSeen,
                 AppData.Configuration.DateTimeViewStringFormat);
 
-            Points = user.Points;
-            Rating = user.Rating;
+            //List<PointsInfo> Points = new List<PointsInfo>(user.Points);
```
Comment (napster1991) [inline Expro/ViewModels/AppUserVM.cs:211]: remove this line

File: Expro/ViewModels/AppUserVM.cs [hunk 4]
```diff
@@ -217,8 +208,9 @@ public AppUserVM(ApplicationUser user)
...
+            //List<RatingsInfo> Ratings = new List<RatingsInfo>(user.Rating);
```
Comment (napster1991) [inline Expro/ViewModels/AppUserVM.cs:212]: remove this line

File: Expro/ViewModels/Expert/ExpertListInfoVM.cs [hunk 5]
```diff
@@ -62,7 +56,7 @@ public string FullName
         public List<string> LawAreasName { get; set; }
         public string ParentLawAreaName { get; set; }
 
-        public string FeedbackRatingStr { get; set; }
+        //public string FeedbackRatingStr { get; set; }
```
Comment (napster1991) [inline Expro/ViewModels/Expert/ExpertListInfoVM.cs:59]: remove this line

File: Expro/ViewModels/Expert/ExpertListInfoVM.cs [hunk 6]
```diff
@@ -76,9 +70,10 @@ public ExpertListInfoVM(ApplicationUser model)
             UserStatusStr = model.UserStatus.Name;
             Email = model.Email;
             IsOnline = model.IsOnline ?? false;
-            Points = model.Points;
-            Rating = model.Rating;
-            FeedbackRating = model.FeedbackRating;
+            RatingsFullInfo = new RatingsFullInfoVM(model.Rating, model.Points, model.FeedbackRating);
+            //Points = model.Points;
```
Comment (napster1991) [inline Expro/ViewModels/Expert/ExpertListInfoVM.cs:74]: remove these lines. comment kod qoldirmaslikka xarakat qilin.

**Review by napster1991 — CHANGES_REQUESTED** [review]

File: Expro/ViewModels/AppUserVM.cs (hunk 2 above)
Comment (KhojiakbarFarkhadov) [reply Expro/ViewModels/AppUserVM.cs]: AppUserVM.cs fileda points bn rating bor faqat feedbackRating property bo'lmagani uchun man alohida PointInfo va RatingsInfo nomli class yartib RatingsFullInfo ni ichida keyn ulani ishlatib ko'rmoqchi edim.

File: Expro/ViewModels/AppUserVM.cs (hunk 1 above)
Comment (KhojiakbarFarkhadov) [reply Expro/ViewModels/AppUserVM.cs]: AppUserVM.cs fileda points bn rating bor faqat feedbackRating property bo'lmagani uchun man alohida PointInfo va RatingsInfo nomli class yartib RatingsFullInfo ni ichida keyn ulani ishlatib ko'rmoqchi edim.

Comment (KhojiakbarFarkhadov) [reply Expro/ViewModels/AppUserVM.cs]: Hozir muammosiz public RatingsFullInfoVM RatingsFullInfo { get; set; } orqali ishlavoti

File: Expro/ViewModels/AppUserVM.cs (hunk 2 above)
Comment (KhojiakbarFarkhadov) [reply Expro/ViewModels/AppUserVM.cs]: Hozir muammosiz public RatingsFullInfoVM RatingsFullInfo { get; set; } orqali ishlavoti

**Review by napster1991 — APPROVED** [review]

**Pull request merged**